src/main/java/relay/entities/AttendanceCode.java

```diff
@@ -0,0 +1,22 @@
+package relay.entities;
+
+public class AttendanceCode {
+    private String code;
+    private String qrCodeImage;
```

Comment:
I think this would probably be need to changed in terms of the type. Like an Image type or smth. First, call the API and see what you get. Then, change the type based on that.